src/chain/elders_info.rs

```diff
@@ -81,8 +81,8 @@ impl EldersInfo {
         &self.members
     }
 
-    pub fn is_member(&self, pub_id: &PublicId) -> bool {
-        self.members.contains_key(pub_id.name())
+    pub fn is_member(&self, name: &XorName) -> bool {
```

Comment:
Commit: Limit the number of in-flight parsec gossip requests 

Considering how often it is used, it is probably a nice quality of life to still have the one taking a PublicId.
Maybe consider renaming `is_member` to `is_member_id`, and adding a new `is_member_name` function (that you can use when you already have the name and from within `is_member_id`.